llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -4781,7 +4781,7 @@ class TargetLowering : public TargetLoweringBase {
   virtual bool CanLowerReturn(CallingConv::ID /*CallConv*/,
                               MachineFunction &/*MF*/, bool /*isVarArg*/,
                const SmallVectorImpl<ISD::OutputArg> &/*Outs*/,
-               LLVMContext &/*Context*/) const
+               LLVMContext &/*Context*/, const Type *RetTy) const
```

Comment:
A new parameter should not be necessary. Outs should already correspond to the decomposed aggregate type derived from the original IR type

Reply:
Because in Mips backend, when process hook function CanLowerReturn, they now use MF function return type to check whether call function return type was origin from fp128. 
In fact, when they check whether MF function return type was origin from fp128, they should use MF function return type; when they check whether call function return type was origin from fp128, they should use call function return type.
So, we need a new parameter to specify the right return type.

Reply:
We can know each element value type in `Outs`, but we can not get the call return type from `Outs`. So I think it is necessary to add a new parameter.